course_discovery/apps/api/serializers.py

```diff
@@ -2315,9 +2343,41 @@ class Meta:
 
 class PersonSearchSerializer(HaystackSerializer):
     profile_image_url = serializers.SerializerMethodField()
+    areas_of_expertise = serializers.SerializerMethodField()
+    urls = serializers.SerializerMethodField()
+    urls_detailed = serializers.SerializerMethodField()
 
     def get_profile_image_url(self, result):
         return result.object.get_profile_image_url
+    
+    def get_areas_of_expertise(self, result):
+        return [{
+            'id': area_of_expertise.id,
+            'value': area_of_expertise.value,
+        } for area_of_expertise in sorted(result.object.areas_of_expertise.all(), key=attrgetter('id'))]
```

Comment:
Do we need to sort here?

Reply:
Broke down the logic in new commit. We need sort here because the response needs to be in order. Avoided using order_by to make additional calls to DB